pkg/cmd/repo/delete/delete.go

```diff
@@ -48,18 +48,21 @@ To authorize, run "gh auth refresh -s delete_repo"`,
 			if len(args) > 0 {
 				opts.RepoArg = args[0]
 			}
+
 			if !opts.IO.CanPrompt() && !opts.Confirmed {
-				return cmdutil.FlagErrorf("--confirm required when not running interactively")
+				return cmdutil.FlagErrorf("--yes or --confirm required when not running interactively")
```

Comment:
Usage instructions should never refer to to deprecated flags. If the purpose of the error message is to teach the user that they have to add a flag, then lets teach them about the flag we want them to use and not mention the deprecated name